### PR TITLE
Add musca_a device type

### PR DESCRIPTION
**Pull request by pfalcon**
This PR is supposed to supercede https://github.com/Linaro/lite-lava-docker-compose/pull/18 and lay the process for adding new device types to our setup in scalable manner.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,5 +50,8 @@ lava-boards:
 	-lavacli -i $(LAVA_IDENTITY) devices add --type qemu --worker lava-dispatcher qemu-01
 	lavacli -i $(LAVA_IDENTITY) devices dict set qemu-01 devices/qemu-01.jinja2
 
+	-lavacli -i $(LAVA_IDENTITY) device-types add musca_a
+	lavacli -i $(LAVA_IDENTITY) device-types template set musca_a device-types/musca_a.jinja2
```
Comment (galak) [inline Makefile]: Should we do this as a while card / loop, and use basename of the file in device-types.

Comment (pfalcon) [reply Makefile]: > Should we do this as a while card / loop, and use basename of the file in device-types.

I'm not sure yet. I may imagine number will grow over time, and if stick to consistent conventions, adding a loop would make sense.

**Comment by galak** [comment]
ok

File: Makefile (hunk 1 above)
Comment (galak) [reply Makefile]: ok

**Pull request merged**